src/components/CheckboxColumn/CheckBoxColumn.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render, fireEvent } from '@folio/jest-config-stripes/testing-library/react';
+import CheckboxColumn from './CheckboxColumn';
+
+describe('CheckboxColumn', () => {
+  const rq = { id: '1' };
+  const selectedRows = {};
+  const toggleRowSelection = jest.fn();
+
+  it('renders correctly', () => {
```

Comment:
Could you rename all your tests cases in all files to follow common pattern? Each test should starts from _"should..."_.